projects/plugins/vaultpress/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"private": true,
+	"description": "VaultPress is a subscription service offering real-time backup, automated security scanning, and support from WordPress experts.",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "pnpm run install-if-deps-outdated && pnpm run clean && pnpm run build-php",
+		"build-js": "echo 'Not implemented.",
+		"build-production": "pnpm run install-if-deps-outdated && pnpm run clean && pnpm run build-production-php",
+		"build-production-js": "echo 'Not implemented.",
+		"build-php": "composer install --ignore-platform-reqs",
+		"build-production-php": "COMPOSER_MIRROR_PATH_REPOS=1 COMPOSER_ROOT_VERSION=dev-master composer install -o --no-dev --classmap-authoritative --prefer-dist",
+		"clean": "rm -rf vendor/",
+		"distclean": "rm -rf node_modules && pnpm run clean",
+		"install-if-deps-outdated": "pnpm install --no-prod --frozen-lockfile",
+		"validate-es5": "eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore"
```

Comment:
I note the package.json doesn't do anything other than define these scripts. We might just move the scripts into composer.json and delete the whole file.

Reply:
Yeah, that makes sense for a PHP-only plugin. is e90ddd3529437ea92ed990041867ebe054fbd898 what you had in mind? I'm not sure we need `COMPOSER_ROOT_VERSION` in there, given https://github.com/Automattic/jetpack/blob/16f7b31b5c57ae218a20f1b6a1c607ea522c9f26/.github/workflows/build.yml#L12 ?

But `COMPOSER_MIRROR_PATH_REPOS` would need to stay, right?

Reply:
The `COMPOSER_MIRROR_PATH_REPOS` needs to stay, yes. It should be ok to leave out the `COMPOSER_ROOT_VERSION`, since hopefully anything actually needing that set is already setting it itself.